pom.xml

```diff
@@ -123,7 +123,17 @@
       </dependencies>
     </profile>
   </profiles>
-
+  <dependencyManagement>
```

Comment:
fixes conflicts between BQ transitive dependencies and gRPC's transitive dependencies